apps/web/app/(auth-pages)/sign-in/page.tsx

```
@@ -21,6 +30,34 @@
 		minLength: 6,
 	})
 
+	useEffect(() => {
```

Biome (1.9.4)

[error] This hook specifies more dependencies than necessary: toast

Outer scope values aren't valid dependencies because mutating them doesn't re-render the component.


(lint/correctness/useExhaustiveDependencies)